src/controllers/exercise_controller.php

```diff
@@ -12,3 +12,12 @@ function createExercise()
 	$exercise = Exercises::create($_POST['exercise_title']);
 	header('Location: /exercises/' . $exercise->getId() . '/fields');
 }
+
+function deleteExercise($id)
+{
+	$exercise = new Exercises($id);
```

Comment:
id the exercises do not exist it will show a stacktrace put a try catch to launch the 404 page (with lost())